sycl/source/detail/device_impl.hpp

```diff
@@ -2313,6 +2325,7 @@ EXPORT_GET_INFO(ext::intel::info::device::current_clock_throttle_reasons)
 EXPORT_GET_INFO(ext::intel::info::device::fan_speed)
 EXPORT_GET_INFO(ext::intel::info::device::min_power_limit)
 EXPORT_GET_INFO(ext::intel::info::device::max_power_limit)
+EXPORT_GET_INFO(ext::intel::info::device::luid)
```

Comment:
This isn't necessary for the new descriptors. This block is guarded with `__INTEL_PREVIEW_BREAKING_CHANGES` at line 2298 and we only need it for the ABI compatibility reasons.

Reply:
Ah ok. I have removed this.